miner/miner.go

```diff
@@ -46,7 +56,11 @@ func NewMiner(api api.FullNode, epp gen.WinningPoStProver, addr address.Address)
 		waitFunc: func(ctx context.Context, baseTime uint64) (func(bool, error), error) {
 			// Wait around for half the block time in case other parents come in
```

Comment:
This comment is a bit misleading, as `PropagationDelaySecs` is 6s.
Just saying.

Reply:
oh yeah, thats been wrong for a while